tests/slack_sdk_async/web/test_web_client_coverage.py

```diff
@@ -17,7 +17,7 @@
 class TestWebClientCoverage(unittest.TestCase):
     # 284 endpoints as of June 12, 2024
     # Can be fetched by running `var methodNames = [].slice.call(document.getElementsByClassName('apiReferenceFilterableList__listItemLink')).map(e => e.href.replace("https://api.slack.com/methods/", ""));console.log(methodNames.toString());console.log(methodNames.length);` on https://api.slack.com/methods
```

Comment:
The list of methods is the output by this JS script. "team.externalTeams.list" is not yet public, so let's hold off making the manual change here until it becomes public.

Reply:
Sounds good 👍